Makefile

```diff
@@ -5,7 +5,9 @@ deps:
 
 test: deps
 	npm run test:all
-	
+	sudo npm install -g validate-maintainers
+	validate-maintainers orbit-db --match --ci
```

Comment:
Since `test` requires `deps`, and `validate-maintainers` is in package.json's `dev-dependencies`, these two lines should be able to be compressed down to simply

```bash
npx validate-maintainers
```

Reply:
Thanks! You're right, this seems to work.